index.html

```diff
@@ -1987,6 +2132,432 @@ <h2>
+          (such as a file browser) if the web app has become the default
+          handler for that file type. It's most useful for custom file types.
+        </aside>
+      </section>
+      <section>
+        <h3>
+          `icons` member
+        </h3>
+        <p>
+          The <a>file handler</a>'s <code><dfn data-dfn-for=
+          "file handler">icons</dfn></code> member lists images that serve as
+          graphical representations of a [=file type=] on a platform. User
+          agents MAY pass this information to the operating system during file
+          handler registration.
+        </p>
+        <aside class="note">
+          Semantically, the [=file handler/icons=] are used to convey the file
+          type itself and also the app. As with [=file handler/name=], this is
+          typically only displayed when the app has become the default handler
+          for a file type.
+        </aside>
+      </section>
+      <section data-cite="MIMETYPE">
+        <h3>
+          `accept` member
+        </h3>
+        <p>
+          The [=file handler=]'s <code><dfn data-dfn-for=
+          "file handler">accept</dfn></code> member is a <a>dictionary</a>
+          mapping [=MIME types=] to a list of [=file extensions=].
+        </p>
+        <p>
+          User agents MUST enforce that the [=file handler/accept=] entry only
+          applies to files that have a matching extension.
+        </p>
+        <p>
+          In order to [=register file handlers=], some operating systems
+          require [=MIME types=] and some require [=file extensions=]. Thus the
+          manifest MUST always provide both for each [=file handler/accept=]
+          entry.
+        </p>
+        <p>
+          In addition to complete [=MIME types=], <code>"*"</code> can be used
+          as the subtype of a MIME type to match, for example, all image
+          formats with <code>"image/*"</code> (that also apply to the provided
+          list of [=file extensions=]).
+        </p>
+        <aside class="note">
+          For example, on Linux both the [=mime type=] and [=file extension=]
+          is used with the <code>xdg-mime</code> utility to specify that the
+          web application can open files with both the mime type specified and
+          the specific file extension. On Windows, only [=file extensions=] are
+          used, so the [=mime types=] are ignored.
+        </aside>
+      </section>
+      <section>
+        <h3>
+          `launch_type` member
+        </h3>
+        <p>
+          The [=file handler=]'s <code><dfn data-dfn-for=
+          "file handler">launch_type</dfn></code> member is a <a>string</a>
+          that determines how the app is launched for files routed to this
+          handler. The possible values are "single-client" and
+          "multiple-clients". If not provided, the default behavior matches
+          "single-client".
```

Comment:
```suggestion
          "multiple-clients". If not provided, it defaults to
          "single-client".
```

Reply:
multiple-clients and single-client should probably link to their definitions and behavior.